src/sentry/api/endpoints/project_stacktrace_link.py

```diff
@@ -246,11 +241,14 @@ def get(self, request: Request, project: Project) -> Response:
             # Post-processing before exiting scope context
             found: bool = result["sourceUrl"] is not None
             scope.set_tag("stacktrace_link.found", found)
-            scope.set_tag("stacktrace_link.auto_derived", derived)
             scope.set_tag("stacktrace_link.source_url", result.get("sourceUrl"))
             scope.set_tag("stacktrace_link.tried_url", result.get("attemptedUrl"))
             if current_config:
                 result["config"] = current_config["config"]
+                scope.set_tag(
+                    "stacktrace_link.auto_derived",
+                    result["config"]["automaticallyGenerated"] is True,
```

Comment:
In the next file we add `automaticallyGenerated` as one of the serialized values.